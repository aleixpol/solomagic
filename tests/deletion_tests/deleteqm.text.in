\ref something0
\ELANParticipant unknown

\mt somethingsomething irena(?) (?) somethingsomething
\ma 
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:13.510
\ELANEnd 00:00:15.512



\ref something1
\ELANParticipant someone

\mt somethingsomething iv(?)enang? somethingsomething
\ma 
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:15.512
\ELANEnd 00:00:18.515

