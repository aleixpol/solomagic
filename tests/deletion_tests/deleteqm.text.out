\ref something0
\ELANParticipant unknown

\mt somethingsomething irena(?) (?) somethingsomething
\ma somethingsomething irena somethingsomething
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:13.510
\ELANEnd 00:00:15.512



\ref something1
\ELANParticipant someone

\mt somethingsomething iv(?)enang? somethingsomething
\ma somethingsomething ivenang? somethingsomething
\mb
\gl 
\ps 

\ft
\nt

\arg
\bwo
\vo

\ELANBegin 00:00:15.512
\ELANEnd 00:00:18.515

